code/WorkInProgress/SomepotatosStuff.dm

```diff
@@ -41,10 +41,21 @@
 			boutput( user, "<span class='notice'>You attach the [CLEAN(src)] to the [CLEAN(what)].</span>" )
 		*/
 
+//#define GPS_MAP_TESTING
```

Comment:
needed? if so, should comment on what it does

Reply:
📝DOCS 📝